cmake/kokkos_arch.cmake

```diff
@@ -107,6 +107,47 @@ KOKKOS_ARCH_OPTION(FIJI            GPU  "AMD Fiji architecture")
 KOKKOS_ARCH_OPTION(VEGA            GPU  "AMD Vega architecture")
 KOKKOS_ARCH_OPTION(GFX901          GPU  "AMD GFX architecture")
 
+
+IF (KOKKOS_ENABLE_CUDA)
+ #Choose default version numbers if not given
+ IF (KOKKOS_ARCH_KEPLER)
+   SET(KOKKOS_ARCH_KEPLER35 ON)
+ ENDIF()
```

Comment:
That does not feel right.  Looking at the generated makefiles I'd say we set `KOKKOS_ARCH_KEPLER` if `KOKKOS_ARCH_KEPLER35` or `KOKKOS_ARCH_KEPLER32` or ... but I wouldn't declare `KOKKOS_ARCH_KEPLER` as an option visible to the user.